README.md

```diff
@@ -69,6 +68,20 @@ It has the following dependancies:
 - yojson
 - redis
 
+### Docker user ? 
+
+You can use the docker to build and run geneweb on any computer. 
+```
+git clone https://github.com/geneweb/geneweb
+cd geneweb 
+docker build -t geneweb .
+docker run --rm -it -p 2316:2316 -p 2317:2317 -v `pwd`:/home/opam/geneweb/ -v YOUR_BASE_DIR:/home/opam/bases/ geneweb /home/opam/geneweb/docker/build.sh --run --api --clean
+```
+the folowing options are available for the build script : 
```

Comment:
the following options are available for the build script:

Reply:
oopsy